src/qtt/instrument_drivers/DistributedInstrument.py

```diff
@@ -42,6 +42,9 @@ def add_measurable_quantity(self, name='quantity', unit='arb.',
                           default_value=default_value, params=params)
         self.add_parameter(name, unit=unit, get_cmd=command,
                            docstring=doc_string)
+    
```

Comment:
According to codacy, "the PR could be better". It's about trailing white space on this line